public/samples/DataStreams/StreamsUpkeep.sol

```diff
@@ -110,65 +98,48 @@ contract StreamsUpkeep is ILogAutomation, StreamsLookupCompatibleInterface {
         return (true, abi.encode(values, extraData));
     }
 
-    function performUpkeep(bytes calldata performData) external override {
+    // function will be performed on-chain
```

Comment:
cannot comment on checkCallback because its not part of the change set, but this comment is for that function on line 94.

We should add comment on top explaning what this does. Eg:
// The Data Streams report bytes is passed here. `extraData` is context data from feed lookup process.
// Your contract may include logic to further process this data.  This method is intended to only be simulated //off-chain by Automation. The data returned will then be passed by Automation into `performUpkeep`

Reply:
Done